packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts

```diff
@@ -117,8 +117,11 @@ export class GithubCreateAppServer {
         ...(this.permissions.includes('members') && { members: 'read' }),
         ...(this.permissions.includes('read') && { contents: 'read' }),
         ...(this.permissions.includes('write') && {
+          administration: 'write',
```

Comment:
Is this only needed for when creating GHApps in personal account or do you need this when creating an org app?

Reply:
I tested it again - it's required for [creating an org repo](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-an-organization-repository) ([administration permission](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#administration))

There is also 3 layers of permissions - these permissions granted are all so-called "Repository Permissions":
![Screenshot 2023-02-10 at 12 14 02](https://user-images.githubusercontent.com/8904624/218078802-b728fcab-6b60-4114-b133-ca35713e41a5.png)

Reply:
Ah now it's all coming back to me.

The reason that this doesn't include it: 
![image](https://user-images.githubusercontent.com/3645856/218080771-a5b7b34c-f8c8-4ff1-9717-653a26e82b1d.png)

Is the fact that you can use the users `github` `oauth` token to create the repository.  I also think that the push is also done as the user too, so maybe it doesn't even need the content and actions.

Maybe we can add it, but say that it's not needed if you use the `github` `oauth` through the `RepoUrlPicker` or something maybe in some docs too.

Reply:
yeah - we don't need the `actions` - will also check for `contents`. It's just a bit frustrating if you setup the GitHub App & expect this to work, but it doesn't :D

My use case was to use the same GHA also for Auth - so adding more context/docs around minimal auth makes sense.